pyproject.toml

```diff
@@ -0,0 +1,17 @@
+[tool.black]
+line-length = 120
+target-version = ['py37', 'py38', 'py39', 'py310']
+include = '\.pyi?$'
+
+[tool.isort]
+profile = 'black'
+src_paths = ['dbt', 'test']
+
+[tool.flake8]
+files = '.*\.py'
+max-line-length = 120
+exclude = ['.git', '.eggs', '__pycache__', 'venv']
+# Error codes to ignore, per file level module imported but unused
```

Comment:
In my $work code, I needed the following ignores to get black compatibility 

```
ignore =
    # space before : (needed for how black formats slicing)
    E203,
    # line break before binary operator (needed for how black formats long lines)
    W503,
```

Reply:
I add those rules.